ICSharpCode.CodeConverter/CSharp/NodesVisitor.cs

```diff
@@ -571,6 +601,24 @@ private BlockSyntax VisitStatements(SyntaxList<VBSyntax.StatementSyntax> stateme
                 return SyntaxFactory.Block(statements.SelectMany(s => s.Accept(methodBodyVisitor)));
             }
 
+            private static bool IsAccessedThroughMyClass(SyntaxNode node, SyntaxToken identifier, ISymbol symbol)
+            {
+                bool accessedThroughMyClass = false;
+                if (symbol.IsVirtual && !symbol.IsAbstract) {
+                    var classBlock = node.Ancestors().OfType<VBSyntax.ClassBlockSyntax>().SingleOrDefault();

```

Comment:
This needs to be `FirstOrDefault()` in case there are nested classes. Need to double check the order too.